NEWS

```diff
@@ -76,6 +76,10 @@ CHANGES WITH 258 in spe:
           option has been deprecated. Also, the 'cryptolib' meson option has
           been deprecated. They will be removed in a future release.
 
+        * systemd-sysusers now unconditionally reads and updates gshadow file
+          when necessary. The meson option 'gshadow' has been deprecated and
+          will be removed in a future release.
```

Comment:
hmm, so we added this in #5920 as gshadow was optional back then apparently, and opensuse opted to disable it? was it actually? is it not anymore?

Reply:
Hm, I did not know that. I will drop the change.